ansible/roles/cli/tasks/deploy.yml

```diff
@@ -3,26 +3,19 @@
 
 - name: "ensure nginx directory for cli exists"
   file:
-    path: "{{ cli_nginx_dir  }}"
+    path: "{{ cli_nginx_dir }}"
     state: directory
 
-- include: docker_login.yml
+- set_fact:
+    cli_installation_mode="{{ openwhisk_cli.installation_mode }}"
```

Comment:
@markusthoemmes In the conditional check downstairs, when: "'{{ openwhisk_cli.installation_mode }}' = 'download'" does not work, but if it has been replaced with the variable, then it works.